servers/src/main/java/tachyon/worker/block/BlockMasterSync.java

```diff
@@ -109,9 +109,9 @@ public void registerWithMaster() throws IOException {
       mWorkerId =
           mMasterClient.worker_register(mWorkerAddress, storeMeta.getCapacityBytesOnTiers(),
               storeMeta.getUsedBytesOnTiers(), storeMeta.getBlockList());
-    } catch (BlockInfoException bie) {
-      LOG.error("Failed to register with master.", bie);
-      throw new IOException(bie);
+    } catch (IOException e) {
```

Comment:
Changes all occurrences of "catch (IOException e)" to "catch (IOException ioe)" in the three files modified by this PR.